src/baselog/api/responses.py

```
@@ -0,0 +1,65 @@
+    request_id: Optional[str] = None
+    timestamp: datetime = field(default_factory=lambda: datetime.now(timezone.utc))
+
+    @classmethod
+    def from_success_response(cls, response):
+        try:
+            json_data = response.json()
+            data = None
+            if json_data is not None:
+                required_keys = {"id", "project_id", "level", "category", "message", "tags", "created_at", "updated_at"}
+                if not all(key in json_data for key in required_keys):
+                    raise ValueError("Invalid response structure")
```

Ruff (0.13.2)

Abstract `raise` to an inner function

(TRY301)